src/dpos/components/OtherDelegations.vue

```diff
@@ -0,0 +1,173 @@
+<template>
+  <div class="validation-delegations py-2">
+    <h6>
+      {{ $t("components.other_delegations.total_staked") }}:
+      {{ totalStaked | tokenAmount(18, 0) }} LOOM
+    </h6>
+    <div role="tablist" v-if="delegations.length">
+      <div v-for="(tier, index) in locktimeTiers">
+        <b-button
+          block
+          aria-controls="collapse-4"
+          @click="showDetail(index)"
+          class="my-2"
+          v-if="getSumDelegations(getDelegationsTier(index)) > 0"
+          v-b-toggle="'accordion-' + currentTier"
+        >
+          <dl>
+            <dt>
+              {{ $t("components.other_delegations.lock_time") }} {{ tier }}
+            </dt>
+            <dd>
+              {{ getSumDelegations(getDelegationsTier(index)) | tokenAmount }}
+              LOOM
+            </dd>
+          </dl>
+        </b-button>
+        <b-collapse
+          :id="'accordion-' + index"
+          class="mt-2"
+          :visible="index == currentTier"
+        >
+          <b-table
+            striped
+            hover
+            :items="getDelegationsTier(index)"
+            :fields="tableFields"
+            :per-page="perPage"
+            :current-page="currentPage"
+          >
+            <template scope="data" slot="delegator">
+              {{ data.item.delegator.local.toString() | loomAddress }}
+            </template>
+            <template scope="data" slot="index">
+              {{ data.item.index }}
+            </template>
+            <template scope="data" slot="amount">
+              {{ data.item.amount | tokenAmount }} LOOM
+            </template>
+            <template scope="data" slot="lockTimeTier">
```

Comment:
Since the delegations are already grouped by tier there's no point displaying this column since the value is going to be the same for all the rows in this table.